core/src/main/java/nl/nn/adapterframework/doc/model/FrankDocModel.java

```diff
@@ -201,9 +277,43 @@ private static Method getParentMethod(String className, String methodName) {
 		}
 	}
 
-	public FrankDocGroup addGroup(String name) {
-		FrankDocGroup group = new FrankDocGroup(name);
-		groups.add(group);
-		return group;
+	private List<ConfigChild> createConfigChildren(	Method[] methods, FrankElement parent) {
```

Comment:
Kan createConfigChildren() (en ook createAttributes()) niet een method van FrankElement zijn?

Reply:
Hier hebben we het daarnet over gehad door de telefoon. Om attributes te creëren moet je @IbisDocRef annotaties parsen en de betreffende FrankElement objecten creëren. Dat zit weer in het model. Je voorstel zou betekenen dat FrankDocModel de class FrankAttribute moet importeren en ook omgekeerd. Dat lukt niet.

Hetzelfde geldt voor configuration children. Om een ConfigChild te maken moet je het ElementType creëren dat erin gaat.